README.md

```diff
@@ -36,14 +49,27 @@ To build the guide locally, take the following steps:
   # Build the book locally!
   $ jupyter-book build .
   ```
+* To view your built book:
 
-  To clean out old pages of your book:
+  Navigate to _build/html/ on your local clone of the repo and open "index.html".
+
+* To clean out old pages of your book (e.g. if you remove a page):
 
   ```
   $ jupyter-book clean .
   ```
 
-## Contributing to this guide
+## Automated build and publishing
+
+Whenever a pull request is opened or changes are pushed to any branch of the base repository, the book is built (separately) by both GitHub Actions and CircleCI. 
+
+### Why both GitHub Actions and CircleCI?
+
+- GitHub Actions is the main build tool. In addition to building whenever a pull request is opened from a fork, it handles publishing the book when changes are made to the main branch (e.g. once the pull request is merged). When that happens, it pushes the built html files to the gh-pages branch, which publishes the book to the [website](https://pyopensci.org/contributing-guide/).
+- CircleCI's build is redundant, but it offers an easier way of viewing the built html in browser WITHOUT merging the changes to main or downloading files. See [How do you preview the the guide from a Pull Request](https://github.com/pyopensci/contributing-guide/#how-do-you-preview-the-guide-from-a-pull-request) below for details. 
+
+### How do you preview the guide from a Pull Request?
```

Comment:
This is awesome. THANK YOU!!

it would be easily to create a litte animated gif of how to preview the files on circle ci potentially using licecap https://licecap.en.softonic.com/ i could do that or if you have time that might be nice to add but not required of course :) i only bring it up because you mentioned images above!!